src/actions/__tests__/literature.test.js

```diff
@@ -0,0 +1,35 @@
+import MockAdapter from 'axios-mock-adapter';
+import { fromJS } from 'immutable';
+
+import { getStoreWithState } from '../../fixtures/store';
+import http from '../../common/http';
+import * as types from '../actionTypes';
+import fetch from '../literature';
+
+const mockHttp = new MockAdapter(http);
+
+const initState = fromJS({
+  loading: false,
+  data: {},
+  error: {},
+});
+
+describe('literature - async action creator', () => {
```

Comment:
there was a problem with the error case right?

Reply:
yep, I'll fix it correctly